src/common/io.py

```diff
@@ -1545,7 +1545,7 @@ def _get_variable_data_as_trajectory(self,
                 name == f'{DIAGNOSTICS_PREFIX}cpu_time'
             ):
             return Trajectory(
-                time, self.get_variable_data(f'{DIAGNOSTICS_PREFIX}cpu_time_per_step').x[start_index:stop_index])
+                time, np.cumsum(self.get_variable_data(f'{DIAGNOSTICS_PREFIX}cpu_time_per_step').x[start_index:stop_index]))
```

Comment:
It looks like I missed the call to `np.cumsum` while refactoring this code before release of PyFMI 2.15:
https://github.com/modelon-community/PyFMI/blob/PyFMI-2.14.x/src/common/io.py#L1452